conf/messages

```diff
@@ -0,0 +1,2 @@
+cookieSettings.title = Cookie settings on HMRC services
+service.name =
```

Comment:
Is something missing here? service.name = what?

Reply:
This suppresses the inclusion of a service name in the GovUK header given it's not service specific - @natcarey is there a better way to do this?